core-tests/src/test/java/org/mule/runtime/core/api/config/FeatureFlaggingRegistryTestCase.java

```diff
@@ -25,12 +25,10 @@
 import org.mule.tck.size.SmallTest;
 
 @SmallTest
-@Feature(DEPLOYMENT_CONFIGURATION)
+@io.qameta.allure.Feature(DEPLOYMENT_CONFIGURATION)
```

Comment:
why?

Reply:
We were referencing two `Feature`s: our enum and the allure class.  After refactoring, we don't need to refer to the enum.